src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -2000,6 +2001,11 @@ oop java_lang_Thread::async_get_stack_trace(oop java_thread, TRAPS) {
       const int max_depth = MaxJavaStackTraceDepth;
       const bool skip_hidden = !ShowHiddenFrames;
 
+      // Pick some initial length
```

Comment:
The comment should at least hint at there being some reasonable reason for choosing the value that follows. :)

Reply:
Sounds good.